crates/bevy_winit/src/lib.rs

```diff
@@ -350,6 +351,35 @@ pub fn winit_runner(mut app: App) {
                         ),
                     }
                 }
+                WindowEvent::DroppedFile(path_buf) => {
+                    let mut cursor_left_events =
```

Comment:
It should be called `dropped_file_events`, right?

Reply:
Indeed. Also fixed this in the other functions.